### ISSUE TITLE
Inaccurate prediction depth

### ISSUE THREAD

**Issue by OPMZZZ** [HKUST-Aerial-Robotics/MVDepthNet#37]
Thank you for your excellent work. Your work is very helpful to me, but I have some problems in depth prediction when using your work. When using example2.py to generate the predicted inverse depth map, although it can effectively distinguish the objects and the background in the scene, there is an estimation error when obtaining the absolute depth of an object. For example, the ground truth is 500mm, but it returns 260mm, while the background is 850mm, but the returned value is 350 mm. Excuse me, what is the problem?

**Comment by OPMZZZ** [comment]
Incidentally, I didn't train my own model, but I used the model you uploaded.

**Comment by OPMZZZ** [comment]
As I am not a major in artificial intelligence, I don't know much about this work. But I used your excellent works in my graduation project, and I am very grateful for your contribution.

**Comment by sunnee1314** [comment]
Hello, can you send me a copy of what you downloaded through the link given by the author? Thank you!

**Comment by sunnee1314** [comment]
> 



> 由于我不是人工智能专业的，所以对这个工作不是很了解。但是我在我的毕业设计中使用了你的优秀作品，我非常感谢你的贡献。

Hello, can you send me a copy of what you downloaded through the link given by the author? Thank you!

**Comment by sunnee1314** [comment]
> As I am not a major in artificial intelligence, I don't know much about this work. But I used your excellent works in my graduation project, and I am very grateful for your contribution.

Hello, can you send me a copy of what you downloaded through the link given by the author? Thank you!And my email address is1465968464@qq.com

**Comment by OPMZZZ** [comment]
Excuse me, what's the link you're talking about?



------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年9月21日(星期三) 下午3:09
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 
Hello, can you send me a copy of what you downloaded through the link given by the author? Thank you!

**Comment by sunnee1314** [comment]
The authors put the two download links above:​the model and&nbsp;the sample images. I can't download them.


叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午3:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 Excuse me, what's the link you're talking about? 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人: ***@***.***&amp;gt;;  
 发送时间: 2022年9月21日(星期三) 下午3:09 
 收件人: ***@***.***&amp;gt;;  
 抄送: ***@***.***&amp;gt;; ***@***.***&amp;gt;;  
 主题: Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37) 
 
 
 
 
 
   
 Hello, can you send me a copy of what you downloaded through the link given by the author? Thank you!

**Comment by sunnee1314** [comment]
Could you send me a copy? Thank you very much! I'm having some problems running the code.

叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午3:12
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 Excuse me, what's the link you're talking about? 
 
 
 
 ------------------&amp;nbsp;原始邮件&amp;nbsp;------------------ 
 发件人: ***@***.***&amp;gt;;  
 发送时间: 2022年9月21日(星期三) 下午3:09 
 收件人: ***@***.***&amp;gt;;  
 抄送: ***@***.***&amp;gt;; ***@***.***&amp;gt;;  
 主题: Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37) 
 
 
 
 
 
   
 Hello, can you send me a copy of what you downloaded through the link given by the author? Thank you!

**Comment by OPMZZZ** [comment]
You are talking about the best model weight trained by the author, whose name is opensource_model.pth.tar?&nbsp; But I don't know what images you're talking about.

**Comment by sunnee1314** [comment]
Yes, I said they were “opensource_model.pth.tar”and“&nbsp;sample_data.pkl.tar.gz”.The other question is: if I want to run this code, I only need to download these two, right?


叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午3:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 You are talking about the best model weight trained by the author, whose name is opensource_model.pth.tar?&amp;nbsp; But I don't know what images you're talking about.

**Comment by OPMZZZ** [comment]
Yes, and the model weights .tar file does not need to be unpacked

**Comment by sunnee1314** [comment]
I have received it. Thank you very much!


叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午3:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 Yes, and the model weights .tar file does not need to be unpacked

**Comment by sunnee1314** [comment]
There is also a question about what the author means by this sentence:
“Please download the model via&nbsp;the link&nbsp;and the sample images via&nbsp;the link. Put the model&nbsp;opensource_model.pth.tar&nbsp;and extract the&nbsp;sample_data.pkl.tar.gz&nbsp;under the project folder.”


叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午3:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 Yes, and the model weights .tar file does not need to be unpacked

**Comment by OPMZZZ** [comment]
This means you need to put those two downloaded files in the same folder as example.py and example2.py，then run example.py

**Comment by sunnee1314** [comment]
Ok, thank you very much!




叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午4:00
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 This means you need to put those two downloaded files in the same folder as example.py and example2.py，then run example.py

**Comment by sunnee1314** [comment]
Thank you. Can you send me a copy?

叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年9月21日(星期三) 下午3:39
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 Yes, and the model weights .tar file does not need to be unpacked

**Comment by OPMZZZ** [comment]
I sent it to you by e-mail before. Didn't you get it?

**Comment by sunnee1314** [comment]
I have received，Thank you very much


叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月11日(星期二) 下午3:05
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 I sent it to you by e-mail before. Didn't you get it?

**Comment by OPMZZZ** [comment]
You are talking about the best model weight trained by the author, whose name is opensource_model.pth.tar?&nbsp; But I don't know what kind of picture you're talking about.

**Comment by sunnee1314** [comment]
I'm sorry. I don't know why I sent you the same message as last time half an hour ago. It's my fault. Thank you for your reply.


叫什么名字好呢
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "HKUST-Aerial-Robotics/MVDepthNet"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年10月11日(星期二) 下午3:29
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [HKUST-Aerial-Robotics/MVDepthNet] Inaccurate prediction depth (Issue #37)





 You are talking about the best model weight trained by the author, whose name is opensource_model.pth.tar?&amp;nbsp; But I don't know what kind of picture you're talking about.

**Comment by denred0** [comment]
@979362835 Hi!
Can you send me the model and images too? I can't download them ( 
espy2005@yandex.ru

Thank you!

**Comment by OPMZZZ** [comment]
What you asked for has been sent to your mailbox.

**Comment by denred0** [comment]
@979362835 Got it. Thank you!

**Comment by JereKnuutinen** [comment]
@979362835  Hi, I would like to get model files also. Thank you.

**Comment by OPMZZZ** [comment]
> @979362835  Hi, I would like to get model files also. Thank you.

You can leave your email address and I will give you the model the next day.

**Comment by juuso-oskari** [comment]
@979362835 Hi, I would like to get model files and samples, since I can't download them through baidu. My email is juuso.o.korhonen@aalto.fi.

**Comment by JereKnuutinen** [comment]
> > @979362835  Hi, I would like to get model files also. Thank you.
> 
> You can leave your email address and I will give you the model the next day.

My email is: teekkari123@gmail.com

**Comment by OPMZZZ** [comment]
> @979362835 Hi, I would like to get model files and samples, since I can't download them through baidu. My email is juuso.o.korhonen@aalto.fi.

I have already sent it.

**Comment by JereKnuutinen** [comment]
Thank you sir!

**Comment by OPMZZZ** [comment]
> > > @979362835  Hi, I would like to get model files also. Thank you.
> 
> > 
> 
> > You can leave your email address and I will give you the model the next day.
> 
> 
> 
> My email is: teekkari123@gmail.com

I have already sent it.

**Comment by juuso-oskari** [comment]
Thanks alot
